src/AccessibilityInsights.SharedUx/Properties/Resources.resx

```diff
@@ -1551,4 +1551,7 @@ Do you want to change the channel? </value>
   <data name="ApplicationSettingsControl_TelemetryDisabledByAdministrator" xml:space="preserve">
     <value>Telemetry has been disabled by an administrator of this computer.</value>
   </data>
+  <data name="A11yElementView_ExecutionError" xml:space="preserve">
+    <value>Execution result failed!</value>
```

Comment:
I don't see the ! in the original string--was that an intentional addition?

Reply:
This was a typo--thanks for catching. Fixed